packages/block-editor/src/store/private-selectors.js

```diff
@@ -543,3 +544,7 @@ export const getBlockStyles = createSelector(
 export function isZoomOutMode( state ) {
 	return state.editorMode === 'zoom-out';
 }
+
+export function getSectionRootClientId( state ) {
+	return state.settings[ sectionRootClientIdKey ] ?? null;
```

Comment:
Rather than return `null` here if it's empty, what about addressing this[ at the source and set it as `null` if there's not a root found](https://github.com/WordPress/gutenberg/blob/add/selector-for-section-root-client-id/packages/editor/src/components/provider/use-block-editor-settings.js#L143-L153)?

Reply:
I think we should remove the default and leave it to `undefined` or whatever is passed in settings.

The "source" is actually just one possible implementation.

Reply:
✅